tests/js/jest.config.js

```diff
@@ -18,6 +18,9 @@ module.exports = {
 		'<rootDir>/build/',
 		'<rootDir>/tests/shared',
 	],
+	modulePathIgnorePatterns: [
+		'<rootDir>/assets/src/components/.*/__mocks__',
```

Comment:
I received the following warning while running the JS tests:

```
jest-haste-map: duplicate manual mock found: index
  The following files share their name; please delete one of them:
    * <rootDir>/assets/src/components/options-context-provider/__mocks__/index.js
    * <rootDir>/assets/src/components/reader-themes-context-provider/__mocks__/index.js
```

It seems Jest doesn't allow mocks with the same base name for some weird reason. It's been filed as [a bug](https://github.com/facebook/jest/issues/2070) for quite some time now.

Ignoring the mocks in `assets/src/components` seems to resolve the issue, but I'd like to get a second opinion from @johnwatkins0 on this.